translate_specs.py

```
@@ -40,7 +40,7 @@ def main() -> None:
 
     path_to_functions = Path(args.functions)
     with path_to_functions.open("rb") as f:
-        c_functions = [cast("ParsecFunction", d) for d in pkl.load(f)]
+        c_functions = [cast("CFunction", d) for d in pkl.load(f)]
```

Ruff (0.14.8)

`pickle` and modules that wrap it can be unsafe when used to deserialize untrusted data, possible security issue

(S301)